frontend/src/App.tsx

```diff
@@ -67,8 +67,11 @@ function App(props: { edit?: boolean }) {
           },
         ];
 
-        if (assistantType === "chat_retrieval" || assistantType === "chatbot") {
-          // The RAG assistant type requires an object with a `messages` field.
+        if (
```

Comment:
This if-statement isn't needed anymore since all chatbots use `StateGraph` with `messages` key.